jest.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  clearMocks: true,
```

Comment:
Same case - can we unify jest config across all repos?

syncing-server-js example: https://github.com/standardnotes/syncing-server-js/blob/develop/package.json#L72-L119

Reply:
I think I can use some of the config from `syncing-server-js` but there is specific config that I need to keep in order for tests to run properly:

- [resetMocks](https://jestjs.io/docs/en/configuration#resetmocks-boolean)
- [restoreMocks](https://jestjs.io/docs/en/configuration#restoremocks-boolean)
- [clearMocks](https://jestjs.io/docs/en/configuration#clearmocks-boolean)

---

As for coverage report, I just use the `html` one. I don't really use the rest.